src/main/java/core/basesyntax/Main.java

```diff
@@ -2,9 +2,10 @@
 
 public class Main {
     public static void main(String[] args) {
-        Lottery lottery = new Lottery();
         for (int i = 0; i < 3; i++) {
-            System.out.println(lottery.getRandomBall());
+            Lottery lottery = new Lottery();
+            System.out.println(lottery.getRandomBall().getColor()
+                    + " " + lottery.getRandomBall().getNumber());
```

Comment:
to print correct info override method toString() in class Ball
```suggestion
            System.out.println(lottery.getRandomBall());
```